### DISCUSSION TITLE
ıbax nedir ? türkish

### DISCUSSION THREAD

**Discussion by hrnaykn** [IBAX-io/go-ibax#854] [Q&A] [answered]
IBAX, benzersiz bir hizmet olarak Blockchain altyapı AĞIDIR. Temel temeli olacak Metaverse NFT madencilik sistemi gelecekteki Metaverse

**Comment by hrnaykn** [comment]
IBAX, benzersiz bir hizmet olarak Blockchain altyapı AĞIDIR. Temel temeli olacak Metaverse NFT madencilik sistemi gelecekteki Metaverse

**Comment by ibaxio** [comment, marked as answer]
The IBAX team is recruiting volunteers to translate the official website. Due to the rapid growth of the Turkish community, Turkish will be uploaded in the next revision.

We also welcome translations from community members in other languages.

Unleash the power of our community and make us stronger.
